index.html

```diff
@@ -324,7 +324,7 @@
           </pre>
           <p>
             The <var>options</var> object contains information about what
-            options the web page wishes to use from the payment request system.
+            options developers wish to use from the payment request system.
```

Comment:
Maybe: "...information the developer needs from the end-user to perform the payment (e.g., the person's name and shipping address)."